platform/broadcom/sai.mk

```diff
@@ -1,6 +1,6 @@
-LIBSAIBCM_XGS_VERSION = 12.3.2.2
+LIBSAIBCM_XGS_VERSION = 12.0.0.8
 LIBSAIBCM_DNX_VERSION = 12.3.2.2
-LIBSAIBCM_XGS_BRANCH_NAME = SAI_12.3.0_GA
+LIBSAIBCM_XGS_BRANCH_NAME = SAI_12.0.0_EA3
```

Comment:
hi @developfast @kperumalbfn , is this the right package?

Reply:
@developfast could you update this to SAI_12.5.0_EA

Reply:
we can probably remove this change. Zhaohui built the bcm package and submitted one for updating the sai package.

Reply:
here is the link: https://github.com/Azure/sonic-buildimage-msft/pull/951.

using a dedicated PR will be better for history tracking.

Reply:
makes sense, we can keep only the buffer related changes in this PR.